libs/web/components/src/ProposalAdvanced.tsx

```diff
@@ -34,18 +37,107 @@ export const ProposalAdvanced: FC<ProposalAdvancedProps> = ({
 			</div>
 			<br />
 			<If condition={open}>
-				<label htmlFor="proposalExtrinsic" className="text-lg">
-					Extrinsic Code
+				<label htmlFor="cennzExtrinsic" className="text-lg">
+					Extrinsic
+				</label>
+				<fieldset
+					id="cennzExtrinsic"
+					className="border-dark mb-4 inline-flex w-full items-center space-x-6 border-[3px] bg-white px-4 py-2"
+				>
+					<p className="mr-2 tracking-widest text-gray-600">api.tx.</p>
+					<Select
+						className="w-[12em] rounded border border-gray-200"
+						inputClassName="cursor-pointer"
+						placeholder="module"
+						defaultValue={proposalCall?.module}
+						onChange={(event: ChangeEvent<HTMLSelectElement>) =>
+							updateProposalCall("module", event.target.value)
+						}
+					>
+						{cennzModules?.map((_module: string) => (
+							<option value={_module}>{_module}</option>
+						))}
+					</Select>
+					<p className="mx-2 tracking-widest">.</p>
+					<Select
+						className="min-w-[12em] rounded border border-gray-200"
+						inputClassName={classNames(
+							proposalCall?.module && "cursor-pointer"
+						)}
+						placeholder="call"
+						defaultValue={proposalCall?.call}
+						onChange={(event: ChangeEvent<HTMLSelectElement>) =>
+							updateProposalCall("call", event.target.value)
+						}
+					>
+						{cennzCalls?.map((call: string) => (
+							<option value={call}>{call}</option>
+						))}
+					</Select>
+				</fieldset>
+				<label htmlFor="cennzValues" className="text-lg">
+					Values
 				</label>
-				<TextArea
-					id="proposalExtrinsic"
-					name="proposalExtrinsic"
-					inputClassName="w-full"
-					value={proposalExtrinsic}
-					onChange={onProposalExtrinsicChange}
-					rows={3}
-				/>
+				<fieldset
+					id="cennzValues"
+					className="border-dark w-full space-y-6 border-[3px] bg-white px-4 py-4"
+				>
+					<If condition={!!extrinsicArgs}>
+						{extrinsicArgs?.map((arg, index) => (
+							<div key={index}>
+								<AutoGrowInput
```

Comment:
Don't think `AutoGrowInput` is necessary here, as it may grow over like this

<img width="1098" alt="Screen Shot 2022-07-03 at 15 37 26" src="https://user-images.githubusercontent.com/97480229/177023543-fd84e7d1-f018-4368-afcc-1607df748375.png">

I'd suggest to have the input in full width, if content is longer than that, it will get cutoff, unless you want to be fancy and turn it into textarea